doc/user/zebra.rst

```diff
@@ -445,7 +465,8 @@ commands in relationship to VRF. Here is an extract of some of those commands:
    This command is available on configuration mode. By default, above command
    permits accessing the vrf configuration mode. This mode is available for
    both VRFs. It is to be noted that *Zebra* does not create *Linux VRF*.
-   Provisioning this command is used to keep the configuration intact.
+   The network administrator can however decide to provision this command in
+   configuration file to get more clarity about its configs deployed.
```

Comment:
`the configuration`
`s/get more clarity about its configs deployed/provide more clarity about the intended configuration`

Reply:
ok. thanks